.gitignore

```diff
@@ -250,3 +250,9 @@ paket-files/
 # JetBrains Rider
 .idea/
 *.sln.iml
+*.txt
+SControlTransportAdapter/Adapter.csproj.DotSettings
```

Comment:
Did you have a project you're not interested to check in? (SControlTransportAdapter)